### PR TITLE
Add zrangebyscore method

### PR DESCRIPTION
**Pull request by dmitrychopey**

### CODE REVIEW

File: lib/protocol/redis/methods/sorted_sets.rb [hunk 1]
```diff
@@ -68,6 +68,24 @@ def zrange(key, start, stop, with_scores: false)
...
+        # @param max [Integer]
+        # @param with_scores [Boolean] Return the scores of the elements together with the elements.
+        # @param limit [Array] Can be used to only get a range of the matching elements (similar to SELECT LIMIT offset, count in SQL).
+        # @example Retrieve the first 10 members with score `>= 0` and `<= 100`
+        #   redis.zrangebyscore("zset", "0", "100", :limit => [0, 10])
+        def zrangebyscore(key, min, max, with_scores: false, limit: nil)
+          arguments = [min, max]
+
+          arguments << 'WITHSCORES' if with_scores
+          arguments.concat(['LIMIT'] + limit) if limit
```
Comment (ioquatix) [inline lib/protocol/redis/methods/sorted_sets.rb]: Do you need to construct a temporary array here?

Comment (dmitrychopey) [reply lib/protocol/redis/methods/sorted_sets.rb]: @ioquatix I've pushed changes without constructing a temporary array.

Comment (ioquatix) [reply lib/protocol/redis/methods/sorted_sets.rb]: Why not just:

```
arguments.push('LIMIT', limit)
```

Comment (ioquatix) [reply lib/protocol/redis/methods/sorted_sets.rb]: Or if limit is an array, 

```
arguments.push('LIMIT', *limit)
```

Comment (dmitrychopey) [reply lib/protocol/redis/methods/sorted_sets.rb]: @ioquatix Good idea. I've also updated other methods in the file in order to be consistent.

Comment (dmitrychopey) [reply lib/protocol/redis/methods/sorted_sets.rb]: @ioquatix Let's merge it? 🚀

**Comment by dmitrychopey** [comment]
@ioquatix Is it possible to bump the gem version, please ?

**Pull request merged**